src/programs/sum_all_eer_files/sum_all_eer_files.cpp

```diff
@@ -0,0 +1,215 @@
+#include "../../core/core_headers.h"
+#include <wx/dir.h>
+
+class
+        SumAllEer : public MyApp {
+
+  public:
+    bool DoCalculation( );
+    void DoInteractiveUserInput( );
+
+  private:
+};
+
+IMPLEMENT_APP(SumAllEer)
+
+// override the DoInteractiveUserInput
+
+void SumAllEer::DoInteractiveUserInput( ) {
+
+    int         new_z_size = 1;
```

Comment:
This variable seems to not be used in DoInteractiveUserInput()?

Reply:
This was copied over from the sum_all_mrc_files program, should I remove it there too?

Reply:
Yes, and thanks - i see you removed in your new commit. Copy and paste is fine as a starting point, but it is good to check that you are using what is there. The codebase is rather large and at times confusing, so the less noise we include, the better imo : )